### PR TITLE
Action 3

### PR DESCRIPTION
**Pull request by anj411**

### CODE REVIEW

File: Api/src/main/java/entpack/api/ApolloBaseApi.java [hunk 1]
```diff
@@ -178,7 +178,44 @@ public boolean createPlayer(String uid, String creditAllocated) {
...
+       paramList.put("x", data);
+
+       String url = HOST + "/Tr_UserInfo.aspx";
+       try {
+           JSONObject obj = postData(url, paramList);
+
+           if (obj != null) {
+               String resp_status = obj.getString("status");
+               if(resp_status != null && resp_status.equals("0000")) {
+                   return obj.get("data");
```
Comment (lakshyaMishraOngraph) [inline Api/src/main/java/entpack/api/ApolloBaseApi.java:207]: return here JSONObject instead of object.

Comment (anj411) [reply Api/src/main/java/entpack/api/ApolloBaseApi.java:207]: I have changed it to   return (JSONObject) obj.get("data");

Comment (anj411) [reply Api/src/main/java/entpack/api/ApolloBaseApi.java:207]: is it right now?

File: Api/src/main/java/entpack/api/ApolloBaseApi.java [hunk 2]
```diff
@@ -178,7 +178,44 @@ public boolean createPlayer(String uid, String creditAllocated) {
...
+               } else {
+                   logger.error("searchPlayer postUrl:{} params:{} result:{}", url, JSONObject.toJSONString(params), obj.toJSONString());
```
Comment (lakshyaMishraOngraph) [inline Api/src/main/java/entpack/api/ApolloBaseApi.java:209]: searchPlayer is not postUrl

File: Api/src/main/java/entpack/api/ApolloBaseApi.java (hunk 1 above)
Comment (lakshyaMishraOngraph) [reply Api/src/main/java/entpack/api/ApolloBaseApi.java:207]: You are fetching only data. But we have to return for now, the things we are getting from api directly in response

File: Api/src/main/java/entpack/api/ApolloBaseApi.java (hunk 2 above)
Comment (anj411) [reply Api/src/main/java/entpack/api/ApolloBaseApi.java:209]: Okay I will change it to GET

File: Api/src/main/java/entpack/api/ApolloBaseApi.java (hunk 1 above)
Comment (anj411) [reply Api/src/main/java/entpack/api/ApolloBaseApi.java:207]: like this:- status:-, data {  }?

**Pull request merged**